app/src/app/rc.js

```diff
@@ -0,0 +1,27 @@
+import path from 'path'
+import {app} from 'electron'
+import {existsSync} from 'fs'
+import loadJson from 'load-json-file'
+import {isWindows, isOSX} from './utils'
+
+// Directory where of .app or .exe file lives.
+let dir
+
+if (isOSX()) dir = path.normalize(app.getAppPath() + '/../../../..')
+else if (isWindows()) dir = path.dirname(app.getPath('exe'))
+else console.warn('Implement rc file path.')
+
+const filePath = path.join(dir, '.graperc')
+
+let config = {}
+
+if (existsSync(filePath)) {
```

Comment:
I could, but I need to log errors which come from json parsing. Otherwise having a typo in the config will just silently fail.